docs/policies.md

```diff
@@ -6,7 +6,321 @@ and [Repository management][2].
+  foo.bar_other:
+    # If the Steering Committee decides to remove a collection for a non-predefined reason.
+    repository: https://github.com/ansible-collections/foo.bar
+    removal:
+      major_version: 20
+      reason: other
+      removal_text: >-
+        Text that must specify why the collection was removed. Can use L(Ansible markup,
+        https://docs.ansible.com/ansible/devel/dev_guide/developing_modules_documenting.html#linking-within-module-documentation).
+      announce_version: 11.2.0
+      discussion: https://forum.ansible.com/t/.../
+```
+
+You can use `antsibull-build lint-build-data --data-dir /path/to/ansible-build-data/X/ X`
+(where `X` is the major version)
+to validate the entries.
+
+### Removing a collection
+
+If a collection should be removed from the upcoming Ansible major version,
+for which a metadata directory already exists but for which feature freeze has not yet happened,
+the collection needs to be removed from the build metadata.
+Also note that the removal should be announced in existing major versions,
+[see the previous section for details on that](#announce-removal-of-a-collection-deprecation).
+
+First, remove the collection's entries from `ansible.in`, `ansible-X.build`, and if necessary, `ansible-X.constraints`.
+
+Then the metadata in `collections-meta.yaml` needs to be updated:
+
+1. For that, locate the collection in the `collections` list.
+   Select the entry, copy it to the clipboard, and remove it from `collections`.
+
+2. Locate the `removed_collections` list and the position where the collection should be inserted.
+   Paste the copied entry from the clipboard.
+
+3. If `removal` already exists, edit it as follows:
+
+     - Replace `major_version` by `version`,
+       and put in the exact Ansible version from which the collection is removed.
+       **Note that this should not happen after the feature freeze!**
+
+     - If `announce_version` is present, remove it.
+       This removes the corresponding changelog entry if the Ansible changelog is regenerated,
+       but that entry should not have been there
+       unless the major version for removal got changed
+       (in that case add an explicit manual entry to `changelog.yaml` if you want to keep the old entry).
+
+4. If `removal` does not yet exist,
+   create it [as described in the previous section](#announce-removal-of-a-collection-deprecation),
+   with the changes as described in the previous point.
+
+As an example, consider the following metadata entry:
+```yaml
+collections:
+  foo.bar:
+    repository: https://github.com/ansible-collections/foo.bar
+    removal:
+      major_version: 20
+      reason: deprecated
+      announce_version: 20.0.0a1
+      discussion: https://forum.ansible.com/t/.../
+```
+This should be moved to `removed_collections` and changed as follows:
+```yaml
+removed_collections:
+  foo.bar:
+    repository: https://github.com/ansible-collections/foo.bar
+    removal:
+      version: 20.0.0a1
```

Comment:
```suggestion
    removal:
      version: 20.0.0a2
```

I think this a confusing. Since you give an example where `announce_version` is 20.0.0a1, the actual removal will happen in a2 or later. Either the upcoming removal is announced in a1 or the collection is removed from a1, but not both... or did I misunderstand something?

Reply:
I've removed the `removal:` block from the 'before' example. If that block is there, then likely with a different major version than the announce version, and the collection is now removed because the major version it should be removed from has been changed. That's something that won't happen too often, so let's better use the more obvious case.